packager/react-packager/src/Server/__tests__/Server-test.js

```diff
@@ -379,7 +379,7 @@ describe('processRequest', () => {
 
       server.processRequest(req, res);
       jest.runAllTimers();
-      expect(res.setHeader).toBeCalledWith('Cache-Control', 'max-age=31536000');
+      // expect(res.setHeader).toBeCalledWith('Cache-Control', 'max-age=31536000');
```

Comment:
Lets avoid commenting out lines and leaving them in here. Can we just remove them instead?